### DISCUSSION TITLE
Is it possible for `s3.to_parquet` to use `partition_cols` as defined in Glue Catalog?

### DISCUSSION THREAD

**Discussion by mattfysh** [aws/aws-sdk-pandas#1644] [Q&A] [answered]
I find that I need to pass in explicit `partition_cols` when calling `s3.to_parquet`, even though these have already been defined in Glue Catalog. Is it possible to make it so that awswrangler grabs the partition columns from Glue (which it already seems to be doing for data types)?

**Comment by kukushking** [comment, marked as answer]
The difficulty here is to distinguish between the case where you would like to create/update the Glue table definition with the new partitioning scheme because atm `wr.s3.to_parquet` keeps table definition in sync with the data, not vice versa. 

In your case I would suggest to use `wr.catalog.get_table()` to retrieve partition keys from the Glue table instead

**Reply by mattfysh** [reply]
Ok cheers, this makes sense, although it would be nice to have an option to avoid multiple `GetTable` calls
